include/mgclient.h

```diff
@@ -129,6 +129,11 @@ extern "C" {
 
 #include <stdint.h>
 
+/// Client software version.
+///
+/// \return Client version in the major.minor.patch format.
+MGCLIENT_EXPORT const char *mg_client_get_version();
```

Comment:
Rename to `mg_get_client_version`.

Reply:
Agree

Reply:
`mg_client_version` is shorter and more consistent with the rest